### PR TITLE
updated default values and added missing npm package

### PR DESCRIPTION
**Pull request by adrianjost**

### CODE REVIEW

File: config/development.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "default": 10,
     "max": 50
   },
-  "mongodb": "MONGO_URI",
+  "mongodb": "http://localhost:27017",
```
Comment (adrianjost) [inline config/development.json]: I'm unsure about this line

**Comment by janrenz** [comment]
Idea: Development Mode should be local dev without docker as in other services

**Comment by janrenz** [comment]
lgtm, lpease await @kremer-io ok before margin

**Comment by janrenz** [comment]
We can make a docker-dev and a stand-alone dev env ..

**Comment by adrianjost** [comment]
I think we solved most of this better and more compatible with #59

**Pull request closed without merging**